tsconfig.json

```diff
@@ -4,8 +4,8 @@
         "module": "commonjs",
         "moduleResolution": "node",
         "isolatedModules": false,
-        "noImplicitAny": true,
-        "strictNullChecks": true,
+        "noImplicitAny": false,
```

Comment:
Hmm...  The `noImplicitAny: true` and `strictNullChecks: true` are not causing any problems on v3 branch or inline-css branch, which was cut from it.  Where did this addCli branch get cut from?